src/BinSkim.Driver.FunctionalTests/BaselineTestsData/NonWindowsExpected/Corrupted_Native_x86_VS2013_Default.exe.sarif

```diff
@@ -0,0 +1 @@
+{"$schema":"http://json.schemastore.org/sarif-1.0.0","version":"1.0.0","runs":[{"id":"22691344-89fa-4fdf-b03c-db1a387980c5","tool":{"name":"BinSkim","fullName":"BinSkim 1.5.0.0","version":"1.5.0.0","semanticVersion":"1.5.0","sarifLoggerVersion":"1.7.1.0","language":"en-US","properties":{"Comments":"A security and correctness analyzer for portable executable and MSIL formats.","CompanyName":"Microsoft","ProductName":"BinSkim Portable Executable Analyzer"}},"results":[],"configurationNotifications":[{"id":"ERR997.ExceptionLoadingAnalysisTarget","physicalLocation":{"uri":"file://Z:/src/BinSkim.Driver.FunctionalTests/BaselineTestsData/Corrupted_Native_x86_VS2013_Default.exe"},"message":"Could not load analysis target 'Corrupted_Native_x86_VS2013_Default.exe'. (Image is either too small or contains an invalid byte offset or count.)","level":"error"}],"invocation":{"startTime":"2018-01-17T00:34:44.418Z","endTime":"2018-01-17T00:34:44.647Z"}}]}
```

Comment:
All tests files will need to be refreshed once we restore default pretty printing. We can ignore issues in this change now, if you are sure you haven't altered analysis results. You can use a private build of SARIF SDK with the pretty-print option restored if it's helpful in the short-term

Reply:
I think this is probably fine for now.  These are all of the non-Windows ones--this change doesn't modify the expected results on Windows (and that test still works correctly), I'll need to do more extensive verification that the results are all correct/expected after the next set of changes.